docs/reference/configuration/Single Sign-On/auth-service.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Protecting a UDS Package with Authservice
+---
+
+To enable authentication for applications that do not have native OIDC configuration, UDS Core can utilize Authservice as an authentication layer.
+
+Follow these steps to protect your application with Authservice:
+
+* Set `enableAuthserviceSelector` with a matching label selector in the `sso` configuration of the Package.
```

Comment:
It might also be good to note in this document that an empty selector (`{}`) can be used to protect all pods in a given namespace.

Reply:
Added clarification. Thanks!